db/database_collection.go

```diff
@@ -256,3 +262,50 @@ func (c *DatabaseCollectionWithUser) User() auth.User {
 func (c *DatabaseCollection) useViews() bool {
 	return c.dbCtx.Options.UseViews
 }
+
+// ////// SYNC FUNCTION:
+
+// Sets the database context's sync function based on the JS code from config.
+// Returns a boolean indicating whether the function is different from the saved one.
+// If multiple gateway instances try to update the function at the same time (to the same new
+// value) only one of them will get a changed=true result.
+func (dc *DatabaseCollection) UpdateSyncFun(ctx context.Context, syncFun string) (changed bool, err error) {
+	if syncFun == "" {
+		dc.ChannelMapper = nil
```

Comment:
Should this actually point back to parent sync function, or is this reinitializing with the default?

Reply:
I don't think we want inheritence on the sync function, but the current RestTester workaround is sort of providing that until we're able to drop that behaviour.